lib/modules.nix

```diff
@@ -515,7 +520,14 @@ rec {
       config.warnings =
         let opt = getAttrFromPath optionName options; in
         optional opt.isDefined
-          "The option definition `${showOption optionName}' in ${showFiles opt.files} no longer has any effect; please remove it.";
+          ''
+            The option definition `${showOption optionName}' in ${showFiles opt.files} no longer has any effect; please remove it.
+
+            ${if replacementInstructions != ""
+              then replacementInstructions
+              else "No instructions were provided on how to replace its functionality."
```

Comment:
This is superfluous; it's not necessary to tell the user we're not telling them something.